modulefiles/build.hera.gnu.lua

```diff
@@ -5,7 +5,7 @@ Load environment to compile UFS_UTILS on Hera using Gnu
 hpss_ver=os.getenv("hpss_ver") or ""
 load(pathJoin("hpss", hpss_ver))
 
-prepend_path("MODULEPATH", "/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.6.0/envs/unified-env/install/modulefiles/Core")
+prepend_path("MODULEPATH", "/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.6.0/envs/unified-env-rocky8/install/modulefiles/Core")
 -- For openmpi:
```

Comment:
@DavidHuber-NOAA - you added this MODULEPATH for openmp at 892b693. If I comment it out, the branch still compiles. Is this line still needed? If so, does it work for Rocky8?

Reply:
@GeorgeGayno-NOAA Good catch.  It seems the SAs have installed a version of `openmpi` in the standard stack, so this line should be removed.

@HenryWinterbottom-NOAA Can you please delete line 10:
```
prepend_path("MODULEPATH", "/scratch1/NCEPDEV/jcsda/jedipara/spack-stack/modulefiles")
```